app/selenium_ui/bamboo/modules.py

```diff
@@ -109,17 +111,19 @@ def view_build_logs(webdriver, datasets):
     def measure():
         view_logs = BuildLogs(webdriver)
         view_logs.view_build_logs()
+        view_logs.wait_for_page_loaded()
 
     measure()
 
 
 def view_job_configuration(webdriver, datasets):
 
-    view_job_configuration_page = JobConfiguration(webdriver, build_plans=datasets['build_plans'])
+    view_job_configuration_page = JobConfiguration(webdriver, build_plans_id=datasets['build_plan_id'])
 
     @print_timing("selenium_view_job_configuration")
     def measure():
         view_job_configuration_page.job_configuration_page()
```

Comment:
You use `go_to_..` naming for actions everywhere, so lets do there `view_job_configuration_page.go_to_job_configuration_page()`